src/Model/Process.php

```diff
@@ -101,7 +93,7 @@ public static function getCode() {
 			return '';
 		}
 
-		return self::$instance->code;
+		return self::$instance;
```

Comment:
Unless the return is type-casted as string, I'm not sure this would have the intended effect. Sure, if it's concatenated in a string, it will work, but it still returns the whole object.